pulseapi/profiles/test_views.py

```diff
@@ -2,12 +2,20 @@
 
 from django.core.urlresolvers import reverse
 
+from .models import UserProfile
+from .views import UserProfileAPIView
+
 from pulseapi.tests import PulseMemberTestCase
 from pulseapi.entries.serializers import EntrySerializer
 from pulseapi.creators.models import OrderedCreatorRecord
 
+from rest_framework.test import APIRequestFactory, force_authenticate
```

Comment:
I don't mind changing to using rest_framework testing tools but it doesn't add too much (only potential request inspection) over the regular django testing tools. For e.g. `force_authenticate` is a small wrapper over `force_login` that django provides and doesn't really change a lot about how it behaves.